.changeset/big-pets-drop.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions': patch
+'@shopify/ui-extensions-react': patch
+---
+
+Added time picking pattern docs
```

Comment:
This will be rolled into the public changelog when this ships as part of a stable version. So, it should be in complete sentences. That being said, I actually don't think you need to add this change log because its just adding docs which doesn't need to be in the changelog. 

For other changes in here though that are adding hooks etc, should have their own change log.

Reply:
> it should be in complete sentences

@jamesvidler Could you help me understand what qualifies as a complete sentence for these change logs? I'm looking at the other ones in `.changeset`, and all the ones I'm seeing so far follow a similar sentence fragment pattern. For example, see `.changeset/dirty-beds-hammer.md` which has "Added details about title/summary truncation to docs" for the patch.

Reply:
You would just run `yarn changeset` multiple times, or manually create the markdown files. But I'd recommend using the command to generate the random names.

> I'm not familiar with the other features that are being rolled in here, so how would you recommend determining how many change logs there should be?

This is a tricky part of this process right now, but what we have done in the past is review the other changes and create ones that make sense. Or, reach out to the authors of those original commits in `checkout-web` and ask them.

To me, these changesets should probably look like:
[ui-extensions-react] [patch]
- Adds `usePaymentMethodAttributes`, `usePaymentMethodAttributeValues`, and `useApplyPaymentMethodAttributeChange` react hooks. 

[ui-extensions] [patch]
- Adds the `bankIdNumber` to the `PaymentOptionItemApi`.
- Adds `background` to `View`, `Grid`, `GridItem`, `Pressable`, `ScrollView`, `InlineStack`,  and `BlockStack` components.